### PR TITLE
Fixed the leak

### PR DESCRIPTION
**Pull request by pnikiel**

### CODE REVIEW

**Comment by pnikiel** [comment]
@ben-farnham this is rather trivial and it'd be great to include it in quasar 1.3.1
OK to merge it?

**Comment by ben-farnham** [comment]
Yep - OK with me. Did you see my comment above? It seemed the more C++11-ish way (to do the same thing).

**Comment by pnikiel** [comment]
I don't see any comment from you apart from the very last one. Could you double check if you published the comment, or re-send it?

File: Server/src/opcserver_open62541.cpp [hunk 1]
```diff
@@ -69,7 +76,7 @@ int OpcServer::start()
 {
     m_server = UA_Server_new(m_server_config);
```
[the remark this answers is not in the document]
Comment (pnikiel) [reply Server/src/opcserver_open62541.cpp]: I will try it out

**Comment by pnikiel** [comment]
Ok this is done. Once I see that AppVeyor build passes I will merge it.

**Pull request merged**